src/components/header.tsx

```diff
@@ -75,10 +75,10 @@ export default function Header() {
           outerWidth,
         )}
       >
-        <Button className="text-md bg-cooper-yellow-500 px-5 py-4 hover:bg-gray-900">
+        <Button className="text-md border-cooper-yellow-500 bg-cooper-yellow-500 px-5 py-4 hover:border-gray-900 hover:bg-gray-900">
```

Comment:
I see you added cooper-yellow-600 so maybe we could use that for the hover styles